createTables.py

```diff
@@ -9,14 +9,14 @@ def createTables():
         "   `site`      varchar(20) NOT NULL,"
         "   `text`      longtext NOT NULL,"
         "   `published` timestamp NOT NULL,"
-        # "   `published` int(11) NOT NULL,"
         "   `ES`        tinyint(1) NOT NULL,"
         "   `US`        tinyint(1) NOT NULL,"
         "   `MX`        tinyint(1) NOT NULL,"
         "   `CO`        tinyint(1) NOT NULL,"
-        "   `type`      varchar(100) NOT NULL,"
+        "   `type`      enum('normal', 'ecommerce', 'slideshow', 'video', 'duplicate', 'branded_club', 'brand_article', 'brand_article_video', 'longform', 'reposted_slideshow') NOT NULL DEFAULT 'normal',"
```

Comment:
i suggest to take it string instead of enum.
adding new types will require table structure to be changed.